muparser-2.2.5/configure

```diff
@@ -3021,13 +3021,13 @@ if test "$ac_test_CFLAGS" = set; then
   CFLAGS=$ac_save_CFLAGS
 elif test $ac_cv_prog_cc_g = yes; then
   if test "$GCC" = yes; then
-    CFLAGS="-g -O2"
+    CFLAGS="-g -O0"
```

Comment:
Do we definitely want this even for release builds?

Reply:
Good catch, I did this for my own debugging. Will remove.